### ISSUE TITLE
Build Failing for Bioconductor

### ISSUE THREAD

**Issue by gabrielodom** [gabrielodom/pathwayPCA#93]
Vignette 5 fails to build.
https://master.bioconductor.org/checkResults/3.12/bioc-LATEST/pathwayPCA/malbec1-buildsrc.html

**Comment by gabrielodom** [comment]
Based on the line numbers, the error is something to do with the `circlize::` function calls. However, I can't replicate the error on my Mac. The error is `argument "y" is missing, with no default`, which doesn't help me very much.

**Comment by gabrielodom** [comment]
I've made some minor changes, and I'm going to commit to see if Travis catches this same issue.

**Comment by gabrielodom** [comment]
So, good news: Travis will show the failing build for Ubuntu: <https://travis-ci.org/github/gabrielodom/pathwayPCA/builds/746833254>. 

@jamesban2015, can you run Vignette 5 in Linux? Here is the message:
```
Quitting from lines 433-481 (Supplement5-Analyse_Results.Rmd) 
Error: processing vignette 'Supplement5-Analyse_Results.Rmd' failed with diagnostics:
argument "y" is missing, with no default
--- failed re-building ‘Supplement5-Analyse_Results.Rmd’
```

I can work on fixing the issue, but I need an idea of what's causing it. I haven't been able to replicate the issue on Mac.

**Comment by jamesban2015** [comment]
@gabrielodom The file "circlePlotData.RDS" cannot be found. Where can I download it?

**Comment by gabrielodom** [comment]
In the vignettes folder: https://github.com/gabrielodom/pathwayPCA/tree/master/vignettes

**Comment by gabrielodom** [comment]
It should come with the package automatically if you clone the repo.

**Comment by jamesban2015** [comment]
@gabrielodom I got the same error when running only the circos plot, and I think here you miss an argument, 
> circos.trackText
function (**sectors, x, y,** labels, track.index = get.current.track.index(), 
    direction = NULL, facing = c("inside", "outside", "reverse.clockwise", 
        "clockwise", "downward", "bending", "bending.inside", 
        "bending.outside"), niceFacing = FALSE, adj = par("adj"), 
    cex = 1, col = par("col"), font = par("font"), factors = sectors) 
I am not sure if that this was changed due to the update of this package

**Comment by gabrielodom** [comment]
I'll try to track that down. Perhaps I don't have the updated version of the `circlize::` package. Can you send me the version number for that package? I'm using 0.4.10. (`Session.info()` will show you.)

**Comment by gabrielodom** [comment]
I just checked Travis. It looks like `circlize::` had an update recently to 0.4.11.

**Comment by gabrielodom** [comment]
I just built vignette 5 with no errors; then I restarted R, installed the newest version of `circlize::`, and built the same vignette. Here is the error:

```
Quitting from lines 433-481 (Supplement5-Analyse_Results.Rmd) 
Error in circos.trackText(rep(-3, nPathway), rep(-3.8, nPathway), labels = "PID_IL1",  : 
  argument "y" is missing, with no default
Calls: <Anonymous> ... suppressMessages -> withCallingHandlers -> circos.trackText
```

**Comment by gabrielodom** [comment]
This should have been closed with commit bd6338f
